applications/matter_bridge/README.rst

```diff
@@ -220,6 +220,46 @@ Adding a simulated bridged device to the Matter bridge
 
       uart:~$ matter_bridge add 256 "Kitchen Light"
 
+Controlling a simulated bridged device
+   Use the following command:
+
+   .. parsed-literal::
+      :class: highlight
+
+      matter_bridge *<"bridged_device_label">* *<new_value>* *<endpoint>*
+
+   In this command:
+
+   * *<"bridged_device_label">* is the label of the simulated Matter bridged device type.
+     The argument is mandatory and accepts the following values:
+
+      * ``onoff`` for On/Off Light.
+      * ``temperature`` for Temperature Sensor.
+      * ``humidity`` for Humidity Sensor.
+
+   * *<new_value>*  is the new integer value that will be generated by the simulated bridged device (note that for ``onoff`` label only *0* and *1* are sensible values).
```

Comment:
can you add information that 0 means off and 1 on?

Reply:
Yes, will add it.